frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -2304,6 +2437,8 @@ mod tests {
 				MultiPhase::feasibility_check(solution, ElectionCompute::Signed),
 				FeasibilityError::UntrustedScoreTooLow,
 			);
+
+			assert_eq!(multi_phase_events(), vec![Event::SignedPhaseStarted { round: 1 }]);
```

Comment:
Tests that don't lead to much event such as this can be skipped.

In general, we mostly want test that reach the call to `elect` to be checked.